.renovaterc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "config:base",
+    ":pinOnlyDevDependencies",
+    "group:monorepos",
```

Comment:
Because `group:monorepos` is included as part of `config:base` (see https://renovateapp.com/docs/config-presets/config-config), it's redundant to include this line, but also completely harmless to leave it in.

Reply:
Are you human or a bot? 😄 . I will leave it in I think.

Reply:
BTW if you ever find that any of your upstream monorepos are *not* being automatically grouped, this is where the definitions originate from: https://github.com/singapore/renovate-config/blob/master/packages/renovate-config-monorepo/generate.js